src/gwt/src/org/rstudio/studio/client/workbench/views/vcs/git/GitStatusRenderer.java

```diff
@@ -78,6 +78,9 @@ public SafeHtml render(String str)
 
    private String descForStatus(String str)
    {
+      if (str.length() < 2)
+         return null;
```

Comment:
nit: previously this function always returned a string; now it's returning `null`? Might be safer to just change the usages of `.charAt()` here.

Reply:
Good catch, meant to return empty string there like one of the cases later in the function. Fixed.